### ISSUE TITLE
Location not available for Android

### ISSUE THREAD

**Issue by sanghyun5958** [Yermo/nativescript-mapbox#345]
I tried to use nativescript-geolocation plugin together.. but not working

**Comment by mikelavigne** [comment]
Same problem. tns add nativescript-geolocation results in an error with nativescript-mapbox plugin.

Error: java.lang.UnsatisfiedLinkError: No implementation found for void com.mapbox.mapboxsdk.net.NativeConnectivityListener.initialize() (tried Java_com_mapbox_mapboxsdk_net_NativeConnectivityListener_initialize and Java_com_mapbox_mapboxsdk_net_NativeConnectivityListener_initialize__)

**Comment by Yermo** [comment]
Android geolocation should be working.  See doGetUserLocation in the demo.

**Comment by mikelavigne** [comment]
Thanks for your reply. I my situation, I need to have access to the geolocation even if no map is displayed so I get the conflict when I try to display the map. I found this [issue](https://github.com/mapbox/mapbox-gl-native/issues/15331) that might be related. The doGetUserLocation in the demo is only using mapbox solution for geolocation. For now I downgraded my nativescript-mapbox package to an old version (4.3.1) and the map displays fine even with the nativescript-geolocation plugin on.

**Comment by Yermo** [comment]
Do you need geolocation data when the app is in the background?

**Comment by mikelavigne** [comment]
No, the app will always be opened. There are various ways around this issue like I could update my location service from mapbox when map is displayed and from the native plugin when it's not, but that would not be very clean for a fix and since both plugins are very popular, I tought it would be better to look into the issue itself.

**Comment by Yermo** [comment]
I need to package up and release my background geolocation plugin which works nicely with this plugin, but i agree that the nativescript-geolocation foreground plugin should work. Unfortunately, I don't have the spare cycles to look into it right now but I will be happy to accept a pull request if you can figure out fix.
